src/attachments/modifyImage.spec.js

```diff
@@ -213,6 +218,102 @@ describe("image modification", () => {
       }));
   });
 
+  it("can resize and trim an image in one step", function () {
```

Comment:
[SUGGESTION] Auch hier ein Vorschlag, eine gemeinsame Test-Funktion zu extrahieren: Ab Zeile 68 unterscheiden sich alle Tests wirklich nur in 1 oder 2 Parametern.